src/markdown/mas/steekproefontwerp_zandleemstreek.Rmd

```diff
@@ -953,37 +953,36 @@ De meest open landschappen zitten volledig binnen de perimeter van het SBP plan.
 De vergelijking van HOL binnen vs. buiten plan en OL binnen vs. buiten plan is dus bemoeilijkt omdat de gemiddelde openheid systematisch hoger zal liggen binnen de SBP perimeter.
 
 ```{r}
-first_samples <- function(steekproef_df, min_n) {
-  min_factor <- min_n / min(allocatie$allocatie)
-  first_design <- data.frame(openheid_klasse = allocatie$openheid_klasse, 
-                             is_sbp = allocatie$is_sbp,
-                             n = ceiling(allocatie$allocatie * min_factor))
+first_samples <- function(steekproef_df, min_n, multiplicatie) {
+  factor_n <- min_n * multiplicatie
   
   steekproef_df %>%
     group_by(openheid_sbp) %>% 
-    mutate(id = row_number()) %>%
+    mutate(id = row_number()) %>% # sample_order cannot be used due to thinning 
     ungroup() %>%
-    full_join(first_design, by = c("openheid_klasse", "is_sbp")) %>%
-    filter(id <= n) %>%
-    select(-n) %>%
-    mutate(puntenset = paste("steekproef minstens", min_n, "per stratum", 
+    filter(id <= factor_n) %>%
+    mutate(batch = ifelse(id <= min_n, batch, "reserve set")) %>%
```

Comment:
Just to be sure: the variable batch is already present in steekproef_df?

Reply:
Yes, in steekproef_df they all have "eerste set" because in the targets pipeline we set `multiplication = 1` (no reserve)